sormas-ui/src/main/java/de/symeda/sormas/ui/location/LocationEditForm.java

```diff
@@ -202,9 +205,26 @@ protected void addFields() {
 		initializeAccessAndAllowedAccesses();
 
 		if (!isEditableAllowed(LocationDto.COMMUNITY)) {
-			setEnabled(false, LocationDto.REGION, LocationDto.DISTRICT);
+			setEnabled(false, LocationDto.COUNTRY, LocationDto.REGION, LocationDto.DISTRICT);
 		}
 
+		if (FacadeProvider.getCountryFacade().getServerCountry() == null) {
```

Comment:
Remove this logic; if no server country is specified, the location forms should behave like they do right now unless a country is selected (in which case everything is disabled of course)